core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -136,6 +136,11 @@ public List<String> getManagerLocations() {
     return context.getManagerLocations();
   }
 
+  @Override
+  public List<String> getScanServers() {
+    return List.copyOf(context.getScanServers().keySet());
+  }
```

Comment:
Why is this returning a list instead of a Set?

Reply:
Resolved in f437fed